### DISCUSSION TITLE
Can't disable omnisharp being auto installed

### DISCUSSION THREAD

**Discussion by marcinjahn** [LazyVim/LazyVim#3761] [Q&A]
Hi,
I want to give csharp-ls a try, but, for some reason, when I disable "omnisharp" and install "csharp-ls", omnisharp gets autoinstalled every time I reopen neovim.

![image](https://github.com/LazyVim/LazyVim/assets/10273406/2a37604a-c7b0-48a3-bede-ddbf14a56678)

I tried to circumvent that by creating a new config of mason-lspconfig in `plugins/mason-lspconfig.lua`:

```lua
return {
  "williamboman/mason-lspconfig.nvim",
  opts = {
    automatic_installation = false,
  },
}
```

That didn't change anything. I have two questions then:

1. Why is omnisharp automatically being installed when I have csharp-ls installed?
2. Why doesn't my override work?

**Comment by dpetka2001** [comment]
Just define in the `nvim-lspconfig` like
```
{
  "neovim/nvim-lspconfig", 
  opts = {
    servers = {
      omnisharp = { enabled = false}, 
    }, 
  }, 
}
```
